.github/pull-request-template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please include a summary of the change and which is is fixed.
+
+- Add A
+- Fix B
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix
+- [ ] New feature
```

Comment:
```suggestion
- New feature
- Maintenance

```
Not checkboxes. They are displayed in PR as progress.

Reply:
> They are displayed in PR as progress.

It's "Type of change", so it's reasonable to be a checkbox. So maintainers  would be able to check quickly. However, in that case asking to delete options is not very reasonable...

Alternatively, we can rewrite that section and ask people to use "Labels" integrated in the GutHub interface. Similarly to "Documentation" I just added to this PR. This also should be easier to integrate with external hooks, I assume 

We can add more labels here https://github.com/Samsung/CredSweeper/labels

Reply:
I mean, checkboxes
 ![image](https://user-images.githubusercontent.com/5725456/187345277-84f6d308-8897-4089-be39-cdca1ea3ddce.png)
Look like 
![image](https://user-images.githubusercontent.com/5725456/187345365-70fc01da-8bf4-42b8-bc10-b0725e20df46.png)
So, there is a confuse that not all checks are passed.

Reply:
Labels are cool, so checkboxes might help not forget add them e.g.:

- [ ] Label for PR assigned
- [ ] Person of PR assigned

Reply:
I think using label is better too.
So I removed `Type of change` section. [ac10b9c](https://github.com/Samsung/CredSweeper/pull/201/commits/ac10b9c4a40180c18f31fab1de1603c4858c8d34)